troubleshooting/known-issues.md

```diff
@@ -6,10 +6,26 @@ Unfortunately, Maestro does not yet have Android support of `inputText` commands
 
 ### \[Android] Accidental double tap
 
-Sometimes, tapOn will try to tap again if it doesn't detect a hierarchy change. To fix such cases, use `retryTapIfNoChange.`  For example:
+Sometimes, tapOn will try to tap again if it doesn't detect a hierarchy change. To fix such cases, use `retryTapIfNoChange`. For example:
 
 ```
 - tapOn:
     text: "Some Button"
     retryTapIfNoChange: false
 ```
+
+### App does not launch
+
+Either the app is not installed or the `appId` is wrong. Note that depending on the app type, the identifier can be different.
+
+To find the appID you can perform the following:
+
+On Android:
+
+1. Execute `adb shell pm list packages` to get a list of all installed packages.
+2. Search for the appID manually or use `grep` in the above command to search for part of the name: `adb shell pm list packages | grep <name>`
+
+On iOS
+
+1. Execute `idb list-targets` to get the `udid` of your simulator.
+2. Execute `idb list-apps --udid <udid>` where you replace `<udid>` with the `udid` from step 1.
```

Comment:
IDB is a good option. However, is not necessarily installed for Maestro CLI users. I believe its better to use something that users might have like _xcrun_.
```suggestion
1. Execute `xcrun simctl listapps booted | grep CFBundleIdentifier` to get a list of all installed packages
2. Search for the appID manually or use `grep` in the above command to search for part of the name: `xcrun simctl listapps booted | grep CFBundleIdentifier | grep <name>`
```